### ISSUE TITLE
Missing support for aria-controls HTML attribute

### ISSUE THREAD

**Issue by adamgian** [adamgian/minify-selectors#37]
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-controls

Note: takes an ID list (space separated)

**Comment by adamgian** [comment]
Addressed in: 555aba1690150ebfee6c894858a838ad56095e30
